src/main/java/org/greenbuttonalliance/gbaresourceserver/usage/model/QualityOfReading.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022 Green Button Alliance, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.greenbuttonalliance.gbaresourceserver.usage.model;
+
+import java.util.EnumSet;
+
+public enum QualityOfReading {
+	VALID(0),
+	MANUALLY_EDITED(7),
+	ESTIMATED_USING_REFERENCE_DAY(8),
+	ESTIMATED_USING_LINEAR_INTERPOLATION(9),
+	QUESTIONABLE(10),
+	DERIVED(11),
+	PROJECTED_FORECAST(12),
+	MIXED(13),
+	RAW(14),
+	NORMALIZED_FOR_WEATHER(15),
+	OTHER(16),
+	VALIDATED(17),
+	VERIFIED(18),
+	REVENUE_QUALITY(19);
+
+	public final int schemaValue;
+
+	QualityOfReading(int schemaValue) {
+		this.schemaValue = schemaValue;
+	}
+
+	public static QualityOfReading getQualityFromSchemaValue(int schemaValue) {
+		return EnumSet.allOf(QualityOfReading.class).stream()
+			.filter(qor -> qor.schemaValue == schemaValue)
+			.findFirst()
+			.orElseThrow(() -> new IllegalArgumentException("No " + QualityOfReading.class.getCanonicalName() + " with schemaValue " + schemaValue));
+	}
+}
```

Comment:
I have these mapped to their integer schema values in DTOs, but if I got too fancy with making this an `enum` instead of just representing them as integers in the database/JPA, let me know and I'll simplify it.

Reply:
These are enum (numeric values) in the schemas with documentation listing the text values.  I like the fact you have converted them to the enumeration string value.